app/api/volunteers/entities.rb

```diff
@@ -12,12 +12,16 @@ class HelpSeeker < Common::Entities::User
     class Need < Common::Entities::Need
       expose :added_by, using: HelpSeeker, expose_nil: true
       expose :reviews, using: Common::Entities::Review, expose_nil: true
+      expose :status_updated_at
+      expose :updated_by
```

Comment:
As mentioned for  the help seekers entities, is better to have them exposed inside common.